recipe/meta.yaml

```diff
@@ -1,18 +1,19 @@
 {% set name = "y-py" %}
-{% set version = "0.6.0" %}
+{% set version = "0.5.9" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/y/y_py/y_py-{{ version }}.tar.gz
-  sha256: 46836169f7dc2957df8513cfe4bc2009175b3a473e630af421a8e75ee1c48f98
+  sha256: 50cfa0532bcee27edb8c64743b49570e28bb76a00cd384ead1d84b6f052d9368
 
 build:
   number: 0
+  skip: True # [s390x]
   script:
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install --no-deps --no-build-isolation . -vv
     - cargo-bundle-licenses --format yaml --output THIRDPARTY.yml
 
 requirements:
```

Comment:
As we do not build cross-compile then these lines can be removed:
```
    - python                                 # [build_platform != target_platform]
    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
    - maturin >=0.13,<0.14                   # [build_platform != target_platform]
```

Reply:
But I suspect that `maturin >=0.13,<0.14` would be required if the build will fail. I saw that when I had been building a rust package with maturin.